src/pylibsshext/channel.pyx

```diff
@@ -113,22 +158,21 @@ cdef class Channel:
         return self.write(data)
 
     def read_bulk_response(self, size=1024, stderr=0, timeout=0.001, retry=5):
-        recv = BytesIO()
         response = b""
```

Comment:
This should be closer to `memoryview`/`buffer` or at least `BytesIO` up until returned for performance. I'll refactor it.